app/page.tsx

```diff
@@ -1,48 +1,205 @@
 import { FiHeart as HeartIcon } from 'react-icons/fi';
 import { GoPaperAirplane as ShareIcon } from 'react-icons/go';
-import { LuUserCircle as ProfileIcon } from 'react-icons/lu';
+//import { LuCircle as ProfileIcon } from 'react-icons/lu';
```

Comment:
You should be using
`import { LuUserCircle as ProfileIcon } from 'react-icons/lu';`
instead of pasting the svg info into the file.